src/sage/algebras/lie_algebras/representation.py

```diff
@@ -94,6 +97,44 @@ def _test_representation(self, **options):
             for v in S:
                 tester.assertEqual(x.bracket(y) * v, x * (y * v) - y * (x * v))
 
+    def representation_matrix(self, elt):
```

Comment:
The method of this name in `sage.modules.with_basis.representation` has additional keyword arguments, including `side`. Should this one too?

Reply:
No, the action is only implemented from the left. So it doesn't make sense to have it.

Reply:
OK then